services/src/datasets/add_from_directory.rs

```diff
@@ -36,9 +37,12 @@ pub async fn add_datasets_from_directory<S: MockableSession, D: DatasetDb<S>>(
         Ok(())
     }
 
-    let dir = fs::read_dir(file_path);
+    let dir = fs::read_dir(&file_path);
     if dir.is_err() {
-        warn!("Skipped adding datasets from directory because it can't be read");
+        warn!(
+            "Skipped adding datasets from directory `{:?}` because it can't be read",
+            file_path
+        );
```

Comment:
Isn't reading from a directory, but cannot access it, more… like an `error!`?

Reply:
8ccc34bde0a3e09e5cd4346a9044f2bc7fbd9535